### PR TITLE
[Media] Distinguish between file and other fields missing.

### PR DESCRIPTION
**Pull request by xlecours**
This is a proposed solution for https://github.com/aces/Loris/issues/7207

> uploading a big (>1024M) file and got an error message saying that there was an upload error. The message said that some of the required fields were missing even though I filled out the PSCID and VLabel.

This adds a specific error message for when the file can't be or is not uploaded.

### CODE REVIEW

File: modules/media/ajax/FileUpload.php [hunk 1]
```diff
@@ -114,7 +114,15 @@ function uploadFile()
     $language   = isset($_POST['language']) ? $_POST['language'] : null;
 
     // If required fields are not set, show an error
-    if (!isset($_FILES, $pscid, $visit)) {
+    if (!isset($_FILES)) {
+        showMediaError(
+            "The uploaded file is missing. Pelase contact the administrator",
```
Comment (nicolasbrossard) [inline modules/media/ajax/FileUpload.php:119]: Pelase => Please

**Review by nicolasbrossard — CHANGES_REQUESTED** [review]

**Comment by nicolasbrossard** [comment]
@xlecours This did not work on my VM. I still get a message telling me that I need to fill all fields, along with an HTTP 400 error inside the JS console. I only filled the PSCID and visit field, along with the file chooser.

**Comment by xlecours** [comment]
`$_FILES` is an empty array when the upload fails so `!isset()` was returning false

**Review by nicolasbrossard — APPROVED** [review]
Works for me. I'd change the error message to "File could not be uploaded successfully. Please contact the administrator": I think the current message seems to indicate that the file field has not been filled out on the form.

File: modules/media/ajax/FileUpload.php [hunk 2]
```diff
@@ -114,7 +114,14 @@ function uploadFile()
     $language   = isset($_POST['language']) ? $_POST['language'] : null;
 
     // If required fields are not set, show an error
-    if (!isset($_FILES, $pscid, $visit)) {
+    if (empty($_FILES)) {
+        showMediaError(
+            "The uploaded file is missing. Please contact the administrator",
```
Comment (xlecours) [inline modules/media/ajax/FileUpload.php:119]:
```suggestion
            "File could not be uploaded successfully. Please contact the administrator.",
```

**Comment by driusan** [comment]
This is failing because of PHPCS

**Comment by xlecours** [comment]
> This is failing because of PHPCS

Is the survey module PR merged?

**Comment by driusan** [comment]
Which survey module PR?

**Comment by xlecours** [comment]
@driusan The one that would fix:
```
FILE: ...work/Loris/Loris/modules/survey_accounts/php/addsurvey.class.inc
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 240 | ERROR | [x] Equals sign not aligned with surrounding
     |       |     assignments; expected 3 spaces but found 2 spaces
 241 | ERROR | [x] Equals sign not aligned with surrounding
     |       |     assignments; expected 2 spaces but found 1 space
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by ridz1208** [comment]
@nicolasbrossard I added the pass manual tests flag because I assumed it was tested given your comment
@xlecours I fixed PHPCS

@driusan in your hands now

**Pull request merged**